Makefile

```diff
@@ -103,10 +103,10 @@ restart: ## restart grafana
 run: \
 	tree
 run: ## start the development server
-	@$(COMPOSE) up -d postgresql
+	@$(COMPOSE) up postgresql
 	@echo "Wait for database to be up..."
 	@$(WAIT_DB)
-	@$(COMPOSE) up -d grafana
+	@$(COMPOSE) up grafana
```

Comment:
Can you explain why?

Reply:
No explanation, it's a mistake for when I was having an issue on my machine